internal/server/http/handler/v1/sensor/handler.go

```diff
@@ -27,14 +28,14 @@ var (
 // @Failure		404	{object}	HTTPError
 // @Failure		500	{object}	HTTPError
 // @Router			/v1/sensor [get]
-// @Param			status	query	string	false	"Sensor Status"
 // @Param			page	query	string	false	"Page"
 // @Param			limit	query	string	false	"Limit"
```

Comment:
Page and limit are integers, right? We can specify types in Swagger
```suggestion
// @Param			page	query	int	false	"Page"
// @Param			limit	query	int	false	"Limit"
```